docs/developing/auth.md

```diff
@@ -42,6 +42,43 @@ related to user identity and permissions:
   `sandstorm-pkgdef.capnp`. The grain's owner holds every permission
   and can use the "Share" button to authorize other users.
 
+* `X-Sandstorm-Preferred-Handle`: The user's preferred "handle". A
+  handle is like a Unix username. It contains only lower-case ASCII
+  letters, digits, and underscores, and it never starts with a digit.
+  The user can set their preferred handle in their account settings.
+  This handle is NOT UNIQUE; it is only a hint from the user. Apps
+  that use handles must decide for themselves whether they need
+  unique handles and, if so, implement some mechanism to deal with
+  duplicates (such as prompting the user to choose a different one,
+  or just appending some digits). Apps should storngly consider
```

Comment:
It will be nice if `sandstorm-http-bridge` can make for per-grain uniqueness. That's still the plan in the near term, as I understand it?

We don't have to tell people that yet.